docs/getting_started/quickstart.rst

```diff
@@ -165,6 +165,11 @@ across a single column via the ``apply`` operation.
 Modin is more than 30X faster at applying a single column of data, operating on 130+ 
 million rows in a second.
 
+In short, Modin provides orders of magnitude speed up over pandas for a variety of operations out of the box. 
```

Comment:
The code used to generate the speedups can be found in `examples/jupyter` at the top-level Modin directory. Would point a link here.

Reply:
The notebook for the quickstart is slightly different from `examples/jupyter` and should be [this one](https://github.com/modin-project/modin/blob/master/examples/quickstart.ipynb). We've linked this on the top of the page for people to follow along.